apheleia-formatters.el

```diff
@@ -1177,7 +1178,7 @@ For more implementation detail, see
     (setq-local indent-line-function
                 (buffer-local-value 'indent-line-function buffer))
     (setq-local lisp-indent-function
-		(buffer-local-value 'lisp-indent-function buffer))
+                (buffer-local-value 'lisp-indent-function buffer))
```

Comment:
Adjusted the indentation in this line to be consistent with the whole file. I can revert it if undesired.